orca-web/src/main/groovy/com/netflix/spinnaker/orca/controllers/AdminController.groovy

```diff
@@ -92,16 +93,18 @@ class AdminController {
   @ResponseStatus(HttpStatus.CREATED)
   Map<String, String> createExecution(@RequestBody Execution execution) {
 
-    if (front50Service && !front50Service.get(execution.application)) {
-      log.warn('No application exists with name: ' + execution.application)
+    // Check if app exists before importing execution.
+    try {
+      front50Service.get(execution.application)
+    } catch(RetrofitError e) {
+      throw new InvalidRequestException("Error received retrieving application: " + execution.application, e)
```

Comment:
Talking to the team, i think it would be better to:
1. Ignore the failures here (e.g. allow importing into an app that doesn't exist - otherwise this EP is not super useful)
2. Add spring config prop to explicitly enable this EP (we would do this on prestaging/test/local only)